### PR TITLE
Manifest v3

### PR DESCRIPTION
**Pull request by KillariDev**
- Adds support for Manifest v3 while still maintaining support for V2

### CODE REVIEW

**Comment by KillariDev** [comment]
Ethers fix is required to get v3 working: https://github.com/ethers-io/ethers.js/issues/3856

File: extension/app/ts/components/App.tsx [hunk 1]
```diff
@@ -185,7 +185,7 @@ export function App() {
 
 	function openAddressBook() {
 		sendPopupMessageToBackgroundPage( { method: 'popup_openAddressBook' } )
-		return window.close() // close extension popup, chrome closes it by default, but firefox does not
+		return globalThis.close() // close extension popup, chrome closes it by default, but firefox does not
```
Comment (0xjimmy) [inline extension/app/ts/components/App.tsx:188]: This needs an issue made?

File: extension/app/manifestV3.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	"permissions": [
+		"activeTab",
+		"storage",
+		"webNavigation",
+		"scripting"
+	],
+	"web_accessible_resources": [
+		{
+			"resources": ["vendor/*", "js/*", "inpage/*"],
+			"matches": ["<all_urls>"] // not sure if this is right?
```
Comment (0xjimmy) [inline extension/app/manifestV3.json:29]: Yes this is valid

**Review by 0xjimmy — APPROVED** [review]
Thats a lot of globalThis

**Pull request merged**